docs/reference/capabilities.md

```diff
@@ -145,6 +145,7 @@ about capabilities, refer to the [Appium documentation](https://appium.io/docs/e
 |`appium:enablePerformanceLogging`| Enable Safari's performance logging (default `false`)| `true`, `false`|
 |`appium:autoWebview`| Move directly into Webview context if available. Default `false`|`true`, `false`|
 |`appium:skipTriggerInputEventAfterSendkeys`| If this capability is set to `true`, then whenever you call the Send Keys method in a web context, the driver will not fire an additional `input` event on the input field used for the call. This event, turned on by default, helps in situations where JS frameworks (like React) do not respond to the input events that occur by default when the underlying Selenium atom is executed. Default `false`|`true`, `false`|
+|`appium:sendKeyStrategy`| If this capability is set to `oneByOne`, then whenever you call the Send Keys method in a web context, the driver will type the given string one by one to the element. This strategy helps in situations where JS frameworks (like React) updates the view for each input. `appium:skipTriggerInputEventAfterSendkeys` functions each input then. Otherwise the driver will type the given string at once to the element. Default is the at once method. |`oneByOne`|
```

Comment:
`appium:skipTriggerInputEventAfterSendkeys` functions each input then`: could you please rephrase/explain this sentence?

Reply:
`Default is the at once method`: what are the other possible values to this capability? Which of them is the default one?